Cartfile.resolved

```diff
@@ -0,0 +1,3 @@
+github "JanGorman/Hippolyte" "0.6.0"
```

Comment:
So, should we have this file in repo or not? We remove it in one PR and add in another

Reply:
@oleksii-lubianyi-epam  It's ok to have it.